test/parallel/test-readline-undefined-columns.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const assert = require('assert');
+const PassThrough = require('stream').PassThrough;
+const readline = require('readline');
+
+// Checks that tab completion still works
+// when output column size is undefined
+
+var iStream = new PassThrough();
+var oStream = new PassThrough();
+
+var rli = readline.createInterface({
+  terminal: true,
+  input: iStream,
+  output: oStream,
+  completer: function(line, cb) {
+
```

Comment:
My bad. I follow the hapi style guide and it's made its way into muscle memory. I'll fix.